From 4e2b2ec2efaf2a98c019377b44185023114f7de7 Mon Sep 17 00:00:00 2001
From: Zhen Chen <chenzhen@rock-chips.com>
Date: Tue, 26 Nov 2019 11:25:09 +0800
Subject: [PATCH] Mali: utgard: adjust corresponding to the refined gpu node in
 dts files

Such as :
	arch/arm/boot/dts/rk312x.dtsi,
	arch/arm/boot/dts/rk322x.dtsi

Change-Id: Ib92193685fc7572558f2f4e65fec6be29d20ac2f
Signed-off-by: Zhen Chen <chenzhen@rock-chips.com>
---
 drivers/gpu/arm/mali400/mali/linux/mali_osk_mali.c | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/drivers/gpu/arm/mali400/mali/linux/mali_osk_mali.c b/drivers/gpu/arm/mali400/mali/linux/mali_osk_mali.c
index c1c099b82d6a..ad8c1723b665 100755
--- a/drivers/gpu/arm/mali400/mali/linux/mali_osk_mali.c
+++ b/drivers/gpu/arm/mali400/mali/linux/mali_osk_mali.c
@@ -217,19 +217,11 @@ uintptr_t _mali_osk_resource_base_address(void)
 	struct resource *reg_res = NULL;
 	uintptr_t ret = 0;
 
-	// reg_res = platform_get_resource(mali_platform_device, IORESOURCE_MEM, 0);
-    /* 
-     * rk_ext : to use dts_for_mali_ko_befor_r5p0-01rel0. 
-     * 关于下面的 "1" : 
-     *      dts_for_mali_ko_befor_r5p0-01rel0 中,
-     *      base_addr 定义在 reg 的 第二个 (index 为 "1") 的 item.
-     */
-	reg_res = platform_get_resource(mali_platform_device, IORESOURCE_MEM, 1);
+	reg_res = platform_get_resource(mali_platform_device, IORESOURCE_MEM, 0);
 
 	if (NULL != reg_res) {
 		ret = reg_res->start;
 	}
-    // D_HEX( (unsigned int)ret);
 
 	return ret;
 }
-- 
2.35.3

